indico/modules/events/api.py

```diff
@@ -341,6 +344,19 @@ def _build_session_api_data(self, session_):
         return [self._serialize_session_block(b, serialized_session, session_access_list, can_manage)
                 for b in session_.blocks]
 
+    @staticmethod
+    def serialize_reference(reference):
```

Comment:
I normally prefer to have this as a simple, function, but it's a matter of personal taste. Just wondering what others think.